src/Umbraco.Cms.ManagementApi/ViewModels/Dictionary/DictionaryOverviewViewModel.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Umbraco.Cms.Core.Models.ContentEditing;
+
+namespace Umbraco.Cms.ManagementApi.ViewModels.Dictionary;
+
+public class DictionaryOverviewViewModel
+{
+    /// <summary>
+    ///     Initializes a new instance of the <see cref="DictionaryOverviewDisplay" /> class.
+    /// </summary>
+    public DictionaryOverviewViewModel() => Translations = new List<DictionaryTranslationOverviewViewModel>();
+
+    /// <summary>
+    ///     Gets or sets the key.
+    /// </summary>
+    public string? Name { get; set; }
+
+    /// <summary>
+    ///     Gets or sets the key.
+    /// </summary>
+    public Guid Key { get; set; }
+
+    /// <summary>
+    ///     Gets or sets the level.
+    /// </summary>
+    public int Level { get; set; }
+
+    /// <summary>
+    ///     Gets or sets the translations.
+    /// </summary>
+    public List<DictionaryTranslationOverviewViewModel> Translations { get; }
```

Comment:
Might just be an oopsie, but there's no setter here, so it doesn't "Gets or sets", it just "Sets" :P